src/classes/BDI_DataImportService.cls

```diff
@@ -89,7 +89,7 @@ global with sharing class BDI_DataImportService {
     */
     public static String strSoqlForDataImportProcess(List<Id> dataImportIds) {
         List<String> whereClauses = new List<String>{
-            'Status__c <> \' + BDI_DataImport_API.bdiImported + \'',
+            'Status__c != \'' + BDI_DataImport_API.bdiImported + '\'',
```

Comment:
Does the similar where clause on line 124 need adjustment as well? 

`'Status__c <> \' + BDI_DataImport_API.bdiImported + \'',`

Reply:
yeah good call. fixing that now